src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
-}
+@import './utils/reset';
+@import './utils/normalize';
+@import './utils/variables';
+@import './utils/colors';
+@import './utils/mixins';
+@import './blocks/catalog';
+@import './blocks/header';
+@import './blocks/stars';
```

Comment:
Ensure that all the imported SCSS files ('reset', 'normalize', 'variables', 'colors', 'mixins', 'catalog', 'header', 'stars') exist in the specified paths. Missing files will cause compilation errors.